### PR TITLE
RSS replies to a user #111

### PR DESCRIPTION
**Pull request by akosourov**
Added `/rss/reply?user=userID&site=siteID` endpoint. It retrieves last 100 comments for site `siteID` and for each comment reads parent to determine is it reply to `userID` or not. Stops iterate when max replies are found or comment were published later than 30 minutes. Skips replies when user replies to himself.
relates to #111

### CODE REVIEW

File: backend/app/rest/api/rss.go [hunk 1]
```diff
@@ -88,6 +89,54 @@ func (s *Rest) rssSiteCommentsCtrl(w http.ResponseWriter, r *http.Request) {
...
 }
 
+// GET /rss/reply?user=userID&site=siteID
+func (s *Rest) rssRepliesCtrl(w http.ResponseWriter, r *http.Request) {
+	userID := r.URL.Query().Get("user")
+	siteID := r.URL.Query().Get("site")
+	log.Printf("[DEBUG] get rss replies to user %s for site %s", userID, siteID)
+
+	data, err := s.Cache.Get(cache.Key(cache.URLKey(r), siteID, userID), func() ([]byte, error) {
+		comments, e := s.DataService.Last(siteID, 100)
```
Comment (umputun) [inline backend/app/rest/api/rss.go]: pls make a const for this, `maxLastForReply` or smth like this

File: backend/app/rest/api/rss.go [hunk 2]
```diff
@@ -88,6 +89,54 @@ func (s *Rest) rssSiteCommentsCtrl(w http.ResponseWriter, r *http.Request) {
...
+		if e != nil {
+			return nil, e
+		}
+		comments = s.adminService.alterComments(comments, r)
+
+		replies := []store.Comment{}
+		for _, c := range comments {
+			if len(replies) > maxRssItems || c.Timestamp.Add(30*time.Minute).Before(time.Now()) {
```
Comment (umputun) [inline backend/app/rest/api/rss.go]: same story as above - MaxReplyMins

File: backend/app/rest/api/rss.go [hunk 3]
```diff
@@ -88,6 +89,54 @@ func (s *Rest) rssSiteCommentsCtrl(w http.ResponseWriter, r *http.Request) {
...
+				break
+			}
+			if c.ParentID != "" && !c.Deleted && c.User.ID != userID { // not interested replies to yourself
+				pc, e := s.DataService.Get(c.Locator, c.ParentID)
```
Comment (umputun) [inline backend/app/rest/api/rss.go]: this made linter sad "app/rest/api/rss.go:111::warning: declaration of "e" shadows declaration at app/rest/api/rss.go:99 (vetshadow)"

**Comment by akosourov** [comment]
Added consts and fixed linter warning

**Pull request merged**